<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterInfo</class>
 <widget class="QDialog" name="RegisterInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>537</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>时间：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>数量：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>130</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>机器码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>180</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>激活码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>671</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>注册信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>240</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改注册信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="label_7">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>激活</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>pushButton_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_9</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_7</zorder>
  <zorder>label_5</zorder>
  <zorder>dateEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
